modules/gdscript/editor/gdscript_highlighter.h

```diff
@@ -104,8 +106,14 @@ class GDScriptSyntaxHighlighter : public EditorSyntaxHighlighter {
 	Color comment_marker_colors[COMMENT_MARKER_MAX];
 	HashMap<String, CommentMarkerLevel> comment_markers;
 
+	// Index 0: paren_level, Index 1: square_bracket_level, Index 2: curly_brace_level
```

Comment:
Style guide.
```suggestion
	// Index 0: paren_level. Index 1: square_bracket_level. Index 2: curly_brace_level.
```
Although the fact that you need this comment tells me this HashMap would be better off using an enum for clarity, even if it's only used here.